stable/prometheus-operator/Chart.yaml

```diff
@@ -11,7 +11,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 6.6.1
+version: 6.6.2
```

Comment:
@dmyerscough since Grafana is going up by a minor version, we should bump the chart minor version too.

Reply:
I bumped the minor version as recommended